src/Tabs/index.jsx

```diff
@@ -89,23 +89,22 @@ class Tabs extends React.Component {
 
     return (
       <div className="tabs">
-        <div
+        <ul
           role="tablist"
           className={classNames([
             'nav',
             'nav-tabs',
           ])}
         >
           {labels}
-        </div>
+        </ul>
         <div role="tabpanel" className="tab-content">
           {panels}
         </div>
       </div>
     );
   }
 }
-
 // TODO: custom validator that ensures labels and panels are the same length
 Tabs.propTypes = {
   labels: PropTypes.oneOfType([
```

Comment:
Let's prefer using `PropTypes.node` over any instance of `PropTypes.oneOfType([PropTypes.string, PropTypes.element])`. In this case we could use: `PropTypes.arrayOf(PropTypes.node)` for labels